cmd/internal/cli/actions.go

```diff
@@ -46,6 +48,12 @@ func getCacheHandle(cfg cache.Config) *cache.Handle {
 	return h
 }
 
+type contextKey int
+
+const (
+	keyOrigImageURI contextKey = iota
```

Comment:
Not clear why you are using an int key here? A string const key would be more usual I think.